flang/include/flang/Lower/CallInterface.h

```diff
@@ -297,6 +297,15 @@ class CallerInterface : public CallInterface<CallerInterface> {
   /// Walk the result non-deferred length specification expressions.
   void walkResultLengths(ExprVisitor) const;
 
+  /// Get the mlir argument that was placed as an actual argument given
+  /// the dummy argument symbol.
+  mlir::Value getPlacedMLIRArgFor(const semantics::Symbol &) const;
```

Comment:
I'm not entirely confident I follow what this method means from this description. I think what this means is

Search the list of arguments to the called function for the `Symbol` (unnamed) and return the Value that is passed in that position in the argument list at this call site.